tests/sentry/replays/test_organization_replay_count.py

```diff
@@ -131,63 +136,77 @@ def test_simple_return_ids(self):
+                "replay": {"replay_id": replay1_id},
             },
-            project_id=self.project.id,
         )
-        self.store_event(
-            data={
-                "event_id": event_id_b,
-                "timestamp": iso_format(self.min_ago),
-                "tags": {"replayId": "z" * 32},  # a replay id that doesn't exist
-                "fingerprint": ["group-1"],
-            },
+        self.create_performance_issue(
             project_id=self.project.id,
+            fingerprint="a",
+            contexts={
+                "trace": {
+                    "trace_id": str(uuid.uuid4().hex),
+                    "span_id": "933e5c9a8e464da9",
+                    "type": "trace",
+                },
+                "replay": {"replay_id": replay3_id},
+            },
         )
-        event_c = self.store_event(
-            data={
-                "event_id": event_id_b,
-                "timestamp": iso_format(self.min_ago),
-                "tags": {"replayId": replay3_id},
-                "fingerprint": ["group-2"],
+        issue2 = self.create_performance_issue(
+            project_id=self.project.id,
+            fingerprint="b",
+            contexts={
+                "trace": {
+                    "trace_id": str(uuid.uuid4().hex),
+                    "span_id": "933e5c9a8e464da9",
+                    "type": "trace",
+                },
+                "replay": {"replay_id": replay2_id},
             },
+        )
+        issue3 = self.create_performance_issue(
             project_id=self.project.id,
+            fingerprint="c",
+            contexts={
+                "trace": {
+                    "trace_id": str(uuid.uuid4().hex),
+                    "span_id": "933e5c9a8e464da9",
+                    "type": "trace",
+                },
+                "replay": {"replay_id": "z" * 32},  # a replay id that doesn't exist
+            },
         )
 
-        query: dict[str, Any] = {
-            "query": f"issue.id:[{event_a.group.id}, {event_c.group.id}]",
-            "returnIds": True,
+        query = {
+            "query": f"issue.id:[{issue1.group.id}, {issue2.group.id}, {issue3.group.id}]",
+            "data_source": Dataset.IssuePlatform,
         }
         with self.feature(self.features):
             response = self.client.get(self.url, query, format="json")
 
         expected = {
-            event_a.group.id: sorted([replay1_id, replay2_id]),
-            event_c.group.id: sorted([replay3_id]),
+            issue1.group.id: 2,
+            issue2.group.id: 1,
         }
         assert response.status_code == 200, response.content
-        self.assertCountEqual(
-            response.data[event_a.group.id],
-            expected[event_a.group.id],
-        )
-        self.assertCountEqual(
-            response.data[event_c.group.id],
-            expected[event_c.group.id],
-        )
+        assert response.data == expected
+
+    def test_invalid_data_source(self):
```

Comment:
can remove this test for now if its not passing

Reply:
Passing now